os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -723,7 +742,7 @@ void up_serialinit(void)
  ****************************************************************************/
 void up_lowputc(char ch)
 {
-	LOGUART_PutChar(ch);
+	LOGUART_PutChar_RAM(ch);
```

Comment:
can't understand why RAM has added as a postfix.

Reply:
Yes please, me too.
Could you help us understand please @AdamZhang0124

Reply:
Apologies for the repetition,
Could you please clarify why the new function is named as "LOGUART_PutChar_**RAM**"?
The significance of the work RAM?